### PR TITLE
➕ESLint

### PR DESCRIPTION
**Pull request by RonjaMeinecke**
https://eslint.org/docs/user-guide/getting-started

• add ESLint `npm install eslint --save-dev`
• set up configuration file `npx eslint --init`
• add .eslintrc.json via questions in terminal:
How would you like to use ESLint? · `problems`
✔ What type of modules does your project use? · `esm`
✔ Which framework does your project use? · `react`
✔ Does your project use TypeScript? · No / `Yes`
✔ Where does your code run? · `browser`, `node`✔  (make sure to choose both via pressing a while choosing)
What format do you want your config file to be in? · `JSON`
• after that install plugins with npm
• save `eslintrc.json` to activate prettier
• add `eslint-config-prettier` to make prettier and ESLint compatible
• add 
import { NextApiRequest, NextApiResponse } from "next";
import "../styles/globals.css";

function MyApp({ Component, pageProps }: AppProps) {
  return <Component {...pageProps} />;
}
to `app.tsx`

• add 
import { NextApiRequest, NextApiResponse } from "next";

export default (req: NextApiRequest, res: NextApiResponse) => {
  res.status(200).json({ name: "John Doe" });
};
to `hello.ts`

### CODE REVIEW

**Review by saschahaselhuhn — APPROVED** [review]
gj

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
+import type { AppProps } from "next/dist/next-server/lib/router/router";
```
Comment (mrana17) [inline pages/_app.tsx]:
```suggestion
import type { AppProps } from "next/dist/next-server/lib/router/router";
```
We used just "next/app" as path

**Review by mrana17 — APPROVED** [review]
lgtm 🚀

File: pages/_app.tsx (hunk 1 above)
Comment (RonjaMeinecke) [reply pages/_app.tsx]: thanks ✌️

**Pull request merged**